xml/ay_firewall.xml

```diff
@@ -341,17 +341,26 @@
    </sect2>
 
    <sect2 xml:id="CreateProfile-firewall-stage">
-      <title>When the Firewall Profile is Applied</title>
+      <title>When the &firewalld; Profile is Applied</title>
 
       <para>
-       The profile is completely applied at the end of the first stage of installation, except when 
-       the installation runs over the network, for example over SSH or VNC. In this case we have 
-       to guarantee that the installed system stays accessible for the whole installation process. For 
-       that reason the firewall configuration is postponed to the second stage. If you need the network 
-       for some reason to fulfill installation successfully, you have to take care of proper firewall 
-       configuration. Last note is for cases when firewall configuration is not present in 
-       the profile at all. Firewall configuration is done according to the product settings as in common
-       installation in such case.
+       The &firewalld; profile is applied at the end of the first 
+       stage of installation. This is an exception to the normal behavior of
+       system configuration occurring in the second stage (see 
```

Comment:
it is not exception. We're moving autoyast configuration into first stage for many (core) modules.   On the other hand, in case of firewall exception is when configuration is postponed to second stage - it happens only if running ssh / vnc installation with second stage enabled (which has to be done explicitly in the profile).

Reply:
So what can happen is:
1) running AY with firewall section, no SSH / VNC -> firewall is configured at the end of 1st stage
2) running AY with firewall section over SSH / VNC and second stage is required -> firewall is configured in the second stage
3) running AY with firewall section over SSH / VNC and no second stage is required -> firewall is configured at the first stage
4) running AY without firewall section -> firewall is configured in the first stage according to product proposals

Reply:
and of course:
5. running AY with or without firewall section together with custom script which requires network access -> firewall is configured in the first stage either according to the profile or the product proposals but the firewall configuration has to be adapted to reflect the script needs (if needed)

Reply:
Thanks for all your work @mchf. Sometimes getting the explanations right is the hardest part of documention. Let me know if this version is correct.

Reply:
sounds good. thanks for giving it right shape.